src/tests/functional/inference_engine/snippets/src/pass/insert_load_store.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include "pass/insert_load_store.hpp"
+#include "common_test_utils/common_utils.hpp"
+#include <subgraph_lowered.hpp>
+
+namespace ngraph {
```

Comment:
namespaxe ov {
namespace test {
namespace snippets {

Reply:
Updated all the namespaces